spec/spec_helper.rb

```
@@ -26,9 +26,14 @@
     rspec.syntax = :expect
   end
 
-  c.before(:all) do
-    OpenAI.configure do |config|
-      config.access_token = ENV.fetch("OPENAI_ACCESS_TOKEN")
+  OpenAI.configure do |config|
+    # to re-generate the recordings you need to have these in your ENV
+    config.access_token = ENV.fetch("OPENAI_ACCESS_TOKEN") do
```

no: fails fast on boot, rather that in the before all